instill/clients/model.py

```diff
@@ -178,6 +202,120 @@ def trigger_model(
             metadata=self.hosts[self.instance].metadata,
         ).send_sync()
 
+    # @grpc_handler
+    # def trigger_async_model(
+    #     self,
+    #     model_name: str,
+    #     task_inputs: list,
+    #     version: str,
+    #     async_enabled: bool = False,
+    # ) -> model_interface.TriggerAsyncUserModelResponse:
+    #     if async_enabled:
+    #         return RequestFactory(
+    #             method=self.hosts[self.instance].async_client.TriggerAsyncUserModel,
+    #             request=model_interface.TriggerAsyncUserModelRequest(
+    #                 name=f"{self.namespace}/models/{model_name}",
+    #                 task_inputs=task_inputs,
+    #                 version=version,
+    #             ),
+    #             metadata=self.hosts[self.instance].metadata,
+    #         ).send_async()
+
+    #     return RequestFactory(
+    #         method=self.hosts[self.instance].client.TriggerAsyncUserModel,
+    #         request=model_interface.TriggerAsyncUserModelRequest(
+    #             name=f"{self.namespace}/models/{model_name}",
+    #             task_inputs=task_inputs,
+    #             version=version,
+    #         ),
+    #         metadata=self.hosts[self.instance].metadata,
+    #     ).send_sync()
```

Comment:
this endpoint still exist, we should keep it

Reply:
is this different from `trigger_model`?

Reply:
reverted